Resources/SEM/qml/MediationAnalysis.qml

```diff
@@ -0,0 +1,154 @@
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+//
+import QtQuick          2.8
+import QtQuick.Layouts  1.3
+import JASP.Controls    1.0
+import JASP.Theme		1.0
+
+
+Form
+{
+    VariablesForm
+    {
+        AvailableVariablesList
+        {
+            name: "availableVariables"
+        }
+        AssignedVariablesList
+        {
+            title: "Predictors"
+            name:  "predictor"
+            allowedColumns: []
+        }
+        AssignedVariablesList
+        {
+            title: "Mediators"
+            name:  "mediators"
+            allowedColumns: ["scale", "ordinal"]
+        }
+        AssignedVariablesList
+        {
+            title: "Outcome"
+            name:  "dependent"
+            allowedColumns: ["scale", "ordinal"]
+        }
+        AssignedVariablesList
+        {
+            title: "Background confounders"
+            name:  "confounds"
+            allowedColumns: []
+        }
+    }
+
+
+	Section
+	{
+        title: qsTr("Options")
+        ColumnLayout {
+            GroupBox
+            {
+                CheckBox { label: qsTr("Standardized estimates") ; name: "std" }
+                CheckBox { label: qsTr("Show lavaan syntax")     ; name: "showSyntax" }
+                CheckBox { label: qsTr("Show R-squared")         ; name: "rsquared" }
+            }
+            GroupBox
+            {
+                title: qsTr("Parameter estimates")
+                CheckBox { label: qsTr("Show direct effects");          name: "showdir";    checked: true }
+                CheckBox { label: qsTr("Show indirect effects");        name: "showind";    checked: true }
+                CheckBox { label: qsTr("Show total indirect effects");  name: "showtotind"; checked: true }
+                CheckBox { label: qsTr("Show total effects");           name: "showtot";    checked: true }
+                CheckBox { label: qsTr("Show residual covariances");    name: "showres";    checked: true }
+            }
+
+        }
+        GroupBox
+        {
+            title: qsTr("Confidence intervals")
+            CIField {
+                text: qsTr("Width")
+                name: "ciWidth"
+            }
+            RadioButtonGroup {
+                title: qsTr("Method")
+                name: "se"
+                RadioButton { text: qsTr("Standard")  ; name: "standard" ; checked: true }
+                RadioButton { text: qsTr("Robust")    ; name: "robust" }
+                RadioButton {
+                    text: qsTr("Bootstrap")
+                    name: "bootstrap"
+                    IntegerField {
+                        text: qsTr("Replications")
+                        name: "bootstrapNumber"
+                        defaultValue: 1000
+                        min: 1
```

Comment:
It might be better to give this a higher minimum, because you run into a host of error messages for the lower numbers